app/src/main/java/org/fossasia/openevent/activities/SessionDetailActivity.java

```diff
@@ -126,4 +122,17 @@ public boolean onCreateOptionsMenu(Menu menu) {
         return super.onCreateOptionsMenu(menu);
     }
 
+    public void createNotification() {
+
+        Calendar calendar = Calendar.getInstance();
+        calendar.setTime(ISO8601Date.getTimeZoneDate(ISO8601Date.getDateObject(session.getStartTime())));
+        calendar.add(Calendar.MINUTE, -10);
```

Comment:
Should probably let the user decide when to be notified via a shared preference etc.